test/chisel2.py

```diff
@@ -39,14 +40,14 @@
 
 def make_client(email=None):
     """Build an acme.Client and register a new account with a random key."""
-    key = jose.JWKRSA(key=rsa.generate_private_key(65537, 2048, default_backend()))
+    key = josepy.JWKRSA(key=rsa.generate_private_key(65537, 2048, default_backend()))
 
-    net = acme_client.ClientNetwork(key, verify_ssl=False,
+    net = acme_client.ClientNetwork(key, acme_version=2,
                                     user_agent="Boulder integration tester")
 
-    client = acme_client.Client(DIRECTORY, key=key, net=net)
+    client = acme_client.Client(DIRECTORY, key=key, net=net, acme_version=2)
     tos = client.directory.meta.terms_of_service
-    if tos is not None and "Do%20what%20thou%20wilt" in tos:
+    if tos is not None and "https://boulder:4431/terms/v7" in tos:
```

Comment:
nit: Can you promote the `"https://boulder:4431/terms/v7"` magic string to a top level var or something that will be easier to spot & change if required? I don't think it deserves to be an env var but it also feels a little brittle burred in the `make_client` function.